docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -39,342 +37,123 @@ Devices with Intel® GNA support:
-|1|3|1|1|1|1|384|240|240|240|240|240|   |   |
-|2|1|1|1|1|1|128|192|192|192|192|192|192|192|
-|2|1|1|1|1|1|256|192|192|192|192|192|192|170|
-|2|1|1|1|1|1|384|192|192|192|192|192|170|113|
-|2|2|1|1|1|1|128|193|193|193|193|193|193|129|
-|2|2|1|1|1|1|256|193|193|193|193|193|193|129|
-|2|2|1|1|1|1|384|193|193|193|193|193|170|113|
-|3|1|1|1|1|1|128|128|128|128|128|128|128|85|
-|3|1|1|1|1|1|256|128|128|128|128|128|128|85|
-|3|1|1|1|1|1|384|128|128|128|128|128|128|85|
-|3|3|1|1|1|1|128|130|130|130|130|87 |   |  |
-|3|3|1|1|1|1|256|130|130|130|130|87 |   |  |
-|3|3|1|1|1|1|384|130|130|130|130|87 |   |  |
-|4|1|1|1|1|1|128|96|96|96|96|96|96|64|
-|4|1|1|1|1|1|256|96|96|96|96|96|96|64|
-|4|1|1|1|1|1|384|96|96|96|96|96|96|64|
-|5|1|1|1|1|1|128|76|76|76|76|76|51|51|
-|5|1|1|1|1|1|256|76|76|76|76|76|51|51|
-|5|1|1|1|1|1|384|76|76|76|76|76|51|51|
-|6|1|1|1|1|1|128|64|64|64|64|64|42|21|
-|6|1|1|1|1|1|256|64|64|64|64|64|42|21|
-|6|1|1|1|1|1|384|64|64|64|64|64|42|21|
-|7|1|1|1|1|1|128|54|54|54|54|54|36|18|
-|7|1|1|1|1|1|256|54|54|54|54|54|36|18|
-|7|1|1|1|1|1|384|54|54|54|54|54|36|18|
-
-
-> **NOTE**:  The above limitations only apply to the new hardware 2D convolution operation.  When possible, the Intel® GNA plugin graph compiler flattens 2D convolutions so that the second generation Intel® GNA 1D convolution operations (without these limitations) may be used.  The plugin will also flatten 2D convolutions regardless of the sizes  if GNA 2.0 compilation target is selected (see below).
-
-## Intel® GNA Forward and Backward Compatibility
+### Intel® GNA Forward and Backward Compatibility
 
 In the general case, there is no guarantee that a model compiled for GNA 2.0 will run on GNA 3.0, or vice versa.
 
-However, in most cases, networks compiled for GNA 2.0 will run as expected on GNA 3.0, although the performance may be worse compared to the case when a network is compiled specifically for the latter.  The exception is networks with convolutions with the number of filters greater than 8192 (see the <a href="#models-and-layers-limitations">Models and Layers Limitations</a> section).
+However, in most cases, networks compiled for GNA 2.0 will run as expected on GNA 3.0, although the performance may be worse compared to the case when a network is compiled specifically for the latter.  The exception is networks with convolutions with the number of filters greater than 8192 (see the <a href="#models-and-operations-limitations">Models and Operations Limitations</a> section).
 
 Networks compiled for GNA 3.0 should run on GNA 2.0 with incompatible layers emulated on CPU.
 
-You can use the following options `KEY_GNA_EXEC_TARGET` and `KEY_GNA_COMPILE_TARGET` options  to check interoperability (see the <a href="#supported-configuration-parameters">Supported Configuration Parameters</a> section below):
+You can use the following options `ov::intel_gna::execution_target` and `ov::intel_gna::compile_target` properties to check interoperability.
```

Comment:
Will it directly return version of GNA or something like that? Should we have code snippet and result here to show how to detect it? For example, there is query_device sample that is referred in GPU docs that returns something - can we use it?

Reply:
@yury-gorbachev Unfortunately, there is no way to see actual target using hello query device sample. Default values of these parameters are empty. Default behaviour depends on 2 things: whether GNA hw is exist, and which latest version is supported by plugin (descrived in Export/Import section). Also we have task  for next releases to prepare instruction for user how to identify GNA device on own laptop/desktop (cc @dorloff ). For current release I added link to Speech Sample where user can play with these options.